porting/first-steps.rst

```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
      repo liblz4-tool bc lzop imagemagick libncurses5 rsync
 
+.. Note::
+    repo is not available in repositories of Ubuntu 20.04 or later versions. Hence it has to be installed manually.
+
+
+Installing repo 
+^^^^^^^^^^^^^^^^^
+
+To install repo, we need to ensure that you curl and git is installed. To install curl & git run the following command :
+
+.. code-block::
+
+    sudo apt install curl && sudo apt install git
+
+Run the following commands to create a .bin directory in the home directory, and include it in your path.
```

Comment:
Mhm ... this is the first time I see the usage of `.bin` with a dot. I'd have a preference for `bin`. In a  quick internet search I find a couple mentions of ``bin``, e.g., https://help.ubuntu.com/community/HomeFolder , but I haven't easily found one with ``.bin``. But, I guess it's not really a standard. Do you feel strongly about the hidden dot.bin folder? If not, pls change it to bin.

Reply:
Corrected as part of next commit